src/dimmwitted.cc

```diff
@@ -165,14 +273,17 @@ void DimmWitted::learn() {
 
   double current_stepsize = opts.stepsize;
   const std::unique_ptr<double[]> prev_weights(new double[nweight]);
+  std::vector<float> delta(nweight);  // 32-bit float
```

Comment:
But that's weird because you're always passing the underlying `.data()` and `.size()` to the packer.  We can also punt on this if you want.